res/database/Default/database_programmes.xml

```diff
@@ -11514,10 +11512,9012 @@ Dream scenes and sentiments warp the storyline.</en>
+				<en>At the end of the 21st century, a group of scientists travels to the edge of the universe to investigate new clues about the origin of all life. What they find there, however, could also mean the downfall of mankind.</en>
+			</description>
+			<staff>
+				<member index="0" function="2">DF-Celebreties-Noomi_Rapace</member>
+				<member index="1" function="2">DF-Celebreties-Michael_Fassbender</member>
+				<member index="2" function="2">DF-Celebreties-Charlize_Theron</member>
+				<member index="3" function="2">DF-Celebreties-Idris_Elba</member>
+				<member index="4" function="1">123df8d5-a8ca-40fe-bdae-89e70d5e0601</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA" year="2012" distribution="1" maingenre="16" subgenre="" flags="0" blocks="3" price_mod="0.67" />
+			<ratings critics="65" speed="41" outcome="64" />
+		</programme>
+
+					<!-- 2015 -->
+
+		<programme id="DF_Filme_Science-Fiction_Der Marsianer - Rettet Mark Watney" product="1" licence_type="1" tmdb_id="0" imdb_id="tt3659388" creator="8782">
+			<title>
+				<de>Der Mars-Indianer - Bringt ihn lebend</de>
+				<en>Save Mark Watney from Mars</en>
+			</title>
+			<title_original>
+				<de>Der Marsianer - Rettet Mark Watney</de>
+				<en>The Martian</en>
+			</title_original>
+			<description>
+				<de>Der Astronaut einer Mars-Mission wird nach einem Sandsturm für tot erklärt und zurückgelassen. Dank seines Einfallsreichtums gelingt es ihm, der Erde sein Überleben zu signalisieren. Die NASA sowie seine heimkehrende Crew starten eine riskante Rettungsaktion.</de>
+				<en>The astronaut of a Mars mission is declared dead after a sandstorm and left behind. Thanks to his ingenuity, he manages to signal his survival to Earth. NASA as well as his crew returning home launch a risky rescue operation.</en>
+			</description>
+			<staff>
+				<member index="0" function="2">DF-Celebreties-Matt_Damon</member>
+				<member index="1" function="2">DF-Celebreties-Jessica_Chastain</member>
+				<member index="2" function="2">0bc5cf6c-b76f-4feb-bfda-ae838f61b317</member>
+				<member index="3" function="2">DF-Celebreties-Kristin_Wiig</member>
+				<member index="4" function="1">123df8d5-a8ca-40fe-bdae-89e70d5e0601</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA" year="2015" distribution="1" maingenre="16" subgenre="1,7" flags="0" blocks="3" price_mod="0.67" />
+			<ratings critics="76" speed="41" outcome="64" />
+		</programme>
+
+		<!-- added (with modifications) from Stukov repo -->
+
+		<programme id="Stukov_Conan, der Barbar" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0082198" creator="9584" created_by="Stukov">
```

Comment:
I'd like to keep IDs "clean" - no spaces or other special characters
Maybe all Stukov programme IDs could be reviewed according to the proposed format.

Reply:
I noticed that too. Fixing.